bindgen/lib.rs

```diff
@@ -127,6 +128,7 @@ bitflags! {
         /// Whether to generate constructors
         const CONSTRUCTORS = 1 << 4;
         /// Whether to generate destructors.
+        /// Whether to generate destructors.
```

Comment:
This looks like a mistaken duplicate line.

Reply:
Thanks! No idea how this happened :upside_down_face: